app/javascript/reducers/messages.js

```diff
@@ -9,16 +9,46 @@ const initialState = {
   items: null,
 }
 
-export default (state : T.HumanOverrideMessagesState, action : T.Action) : T.HumanOverrideMessagesState => {
+export default (state: T.HumanOverrideMessagesState, action: T.Action): T.HumanOverrideMessagesState => {
   state = state || initialState
   switch (action.type) {
     case actions.FETCH: return fetch(state, action)
     case actions.RECEIVE: return receive(state, action)
-    case actions.ANSWER: return answerMessage(state, action)
+    case actions.ANSWER_SUCCESS: return answerSuccess(state, action)
+    case actions.RESOLVE_SUCCESS: return resolveSuccess(state, action)
     default: return state
   }
 }
 
+const resolveSuccess = (state, {messageId}) => {
+  return {
+    ...state,
+    items: omit(state.items, [messageId]),
+  }
+}
+
+const answerSuccess = (state, action) => {
+  const { messageId, answer } = action
+  const { items } = state
+
+  if (items) {
+    const message = { ...items[`${messageId}`] }
```

Comment:
This string interpolation doesn't feel right, but it was the only way to get rid of this error: 
![image](https://user-images.githubusercontent.com/39921597/45306178-68e93780-b4f2-11e8-8da8-81c0aa28f46c.png)